runtests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+export FLASK_ENV="TESTING"
+export PYTHONPATH=$(pwd)  # setting project root as PYTHONPATH
+./manage.py init  # creates the test upload directory
```

Comment:
Move this line to `.travis.yml`, It's part of the install steps, not the test running steps.